bedwars-plugin/src/main/java/com/tomkeuper/bedwars/listeners/DamageDeathMove.java

```diff
@@ -306,15 +307,16 @@ public void onDeath(PlayerDeathEvent e) {
         Player victim = e.getEntity(), killer = e.getEntity().getKiller();
         ITeam killersTeam = null;
         IArena a = Arena.getArenaByPlayer(victim);
+
+        if (a == null) return;
+
         Player bedDestroyer = a.getTeam(victim).getBedDestroyer();
 
         if ((BedWars.getServerType() == ServerType.MULTIARENA && BedWars.getLobbyWorld().equals(victim.getWorld().getName())) || a != null) {
             e.setDeathMessage(null);
         }
 
 
-        if (a == null) return;
-
```

Comment:
Wouldn't the previous position be useless and only cause problems with NPE? I'm not exactly sure with why it was there.